contracts/common/bases/StrategyBaseUpgradeable.sol

```diff
@@ -22,7 +23,7 @@ struct RoleToUsers {
 
 struct StrategyArgs {
     IInvestmentToken investmentToken;
-    IERC20Upgradeable depositToken;
+    IERC20UpgradeableExt depositToken;
```

Comment:
Isn't this make `getDepositToken` returns `IERC20UpgradeableExt`? If so, L13 and L52 is not used anymore.

+ Same thing goes to PortfolioBase as well.

Reply:
in most OO languages, if you have:

function f() returns (A a), and you have a class B which derives/extends A, then you can always return an object of class B

Reply:
I know. So IERC20UpgradeableExt is used only inside this project?

Reply:
I think it is fine to restrict IERC20UpgradeableExt to this file now. We can change it later and return IERC20UpgradeableExt, if it is really needed. However .decimal usage should be minimized(as it is non standard), so that is why I am a bit hesitant to return it from other function.